documentation/src/docs/admin-guide/num-containers.md

```diff
@@ -0,0 +1,89 @@
+
+- _Compute resources per Segment Store_: Each Segment Container runs a set of services that consume CPU power. 
+This means that the number of Segment Containers in the system is limited by the available compute power in the cluster.
+As a rule of thumb, we can define a number of Segment Containers per Segment Store according to the number of cores
+per instance (e.g., 1 or 2 Segment Containers per core, depending on the core power). Similarly, the size of the thread
+pool (`pravegaservice.threadPool.core.size`) should be sized according to the number of segment containers 
+(e.g., 2-4 threads per Segment Container, with a minimum number of 20 threads). Note that setting a value of Segment
+Containers and/or threads higher than necessary may lead to performance degradation due to CPU contention under
+intense workloads.
+
+- _Initial number of Segment Stores_: This deployment decision plays a central role when it comes to define the number 
+of Segment Containers in the system and may vary depending on the deployment scenario. On the one hand, an on-premise 
+environment we may have a fixed amount of hardware available to run Pravega, in which case the number of Segment Store 
+instances may be known. On the other hand, in a cloud environment a reasonable approach would be to right-size the initial 
+number of Segment Store instances based on the expected load. We have published performance analysis that can help users 
+to understand the load that a Pravega Cluster can sustain, as well as the associated latency numbers 
+[[1]](https://blog.pravega.io/2020/10/01/when-speeding-makes-sense-fast-consistent-durable-and-scalable-streaming-data-with-pravega/), 
+[[2]](https://blog.pravega.io/2021/03/10/when-speed-meets-parallelism-pravega-performance-under-parallel-streaming-workloads/). 
+In addition to that, we also provide benchmark tools that can help users to generate loads that resemble that of users
+to infer the right number of Segment Store instance that can reasonably absorb a given workload
+(e.g., [Pravega Benchmark](https://github.com/pravega/pravega-benchmark), 
+[OpenMessagin Benchmark Pravega Driver](https://github.com/pravega/openmessaging-benchmark)). We also have developed
+a [provisioner tool for Pravega](https://github.com/pravega/pravega-tools/tree/master/pravega-provisioner) to help you 
+reason about these aspects.
+
+With the previous points in mind, we can define the number of Segment Containers in the system 
+(i.e., `pravegaservice.container.count`, `controller.container.count`) by just multiplying the number of Segment Containers 
+per instance by the number of initial instances. Let's see a practical example. Image that we want to deploy a Pravega 
+Cluster using AWS EKS instances with 8 vCPUs and 16GB of RAM for running Segment Stores. If we consider the normal load
+in the system to be 50MBps (1KB events), then we can safely assume that 3 Segment Stores may be enough to absorb it.
```

Comment:
I'm concerned that if we say `50MBps`, someone may use that to say "Pravega can only do 50MBps with 8 vCPUs and 16GB of RAM" - which is clearly false. Is there a way we can rephrase these paragraphs without mentioning (hypothetical) speeds?

Reply:
The paragraph attempts to be hypothetical: 
`Image that we want to deploy a Pravega...`
I have also replaced `consider` by `assume` in the sentence below.